(Doc+) link video for resolving shards too large (#114915)
elastic/elasticsearch: docs/reference/how-to/size-your-shards.asciidoc

<<indices-add-alias,Create Alias>> against the destination index for the source 
index's name to point to it for continuity. 

> See link:https://www.youtube.com/watch?v=sHyNYnwbYro[this video] for example

Reviewer: ```suggestion
See this https://www.youtube.com/watch?v=sHyNYnwbYro[fixing shard sizes video] for an example troubleshooting walkthrough.
```

@stefnestor two minor remarks:

- I'd like to remove `link:` because it's superfluous
- I'd prefer for the link text to be a better indicator of the video content. (This comment applies to your other PR too 🙏 )

The descriptive link text is better for SEO and A11Y (screen readers).
This will make it easier to update/remove these in the future if this content becomes stale or unavailable.

> troubleshooting walkthrough.

Author: ```suggestion
```

<<indices-add-alias,Create Alias>> against the destination index for the source 
index's name to point to it for continuity. 

See this https://www.youtube.com/watch?v=sHyNYnwbYro[fixing shard sizes video] for an example troubleshooting walkthrough.

> troubleshooting walkthrough.

Reviewer: ```suggestion
```

oops my bad for the trailing text
